app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -24,22 +24,21 @@ def current_intake
       end
 
       def question_navigator
-        case params[:us_state]
-        when 'az'
-          Navigation::StateFileAzQuestionNavigation
-        when 'ny'
-          Navigation::StateFileNyQuestionNavigation
+        @navigator ||= "Navigation::StateFile#{state_code.titleize}QuestionNavigation".constantize
```

Comment:
Using this is easier than enumerating out states individually - especially as we add more states